modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/jaxrs/param/converter/provider/SiteParamConverterProvider.java

```diff
@@ -113,6 +105,33 @@ public String toString(Long parameter) {
 		return String.valueOf(parameter);
 	}
 
+	private boolean _containsValue(
+		MultivaluedMap<String, String> multivaluedMap, String key,
+		String value) {
+
+		List<String> values = multivaluedMap.get(key);
+
+		if ((values != null) && values.contains(value)) {
+			return true;
+		}
+
+		return false;
+	}
+
+	private Long _getGroupId(
+		MultivaluedMap<String, String> multivaluedMap, String parameterValue) {
+
+		if (_containsValue(multivaluedMap, "assetLibraryId", parameterValue)) {
```

Comment:
I don't understand this changes, because we are using assetLibraryId and siteId as assetLibraryKey and siteKey

Reply:
That is an existing behavior; I'm just making it work when both assetLibraryId and siteId are present.

Reply:
ok, that's fine then